biahub/cli/disk.py

```diff
@@ -0,0 +1,125 @@
+import os
+import shutil
+import subprocess
+
+from datetime import datetime
+from pathlib import Path
+
+import click
+
+from humanize import naturalsize
+
+
+def get_dir_size_du(path: str) -> int:
+    """
+    Use `du -sb` to get the total size in bytes of a directory or file.
+    Follows symlinks to measure the actual data.
+    """
+    resolved_path = Path(path).resolve()
+    if not resolved_path.exists():
+        raise FileNotFoundError(
+            f"[get_dir_size_du] Path does not exist: {resolved_path.as_posix()}"
+        )
+
+    try:
+        result = subprocess.run(
+            ['du', '-sb', resolved_path.as_posix()], capture_output=True, check=True, text=True
+        )
+        size_bytes = int(result.stdout.strip().split()[0])
+        return size_bytes
```

Comment:
As an aside, `du -sb` only works on Linux, while on macOS `b` is an invalid option. Linux uses GNU `du`, macOS uses BSD `du`